src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -489,46 +509,6 @@ def build(
             placeholder_query=placeholder_query,
         )
 
-    @SnowflakePlan.Decorator.wrap_exception
-    def build_from_multiple_queries(
```

Comment:
@sfc-gh-aalam this function seems not used anymore after you remove the create_temp_table, I removed it here because it was complaining i don't have test coverage for this code.

Reply:
thanks